src/components/collapsible-section/collapsible-section.tsx

```
@@ -147,6 +154,7 @@ export class CollapsibleSection {
                     class="body"
                     aria-hidden={String(!this.isOpen)}
                     id={this.bodyId}
+                    role="region"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<section>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)